pkg/sql/parser/builtins.go

```diff
@@ -1442,18 +1442,22 @@ var Builtins = map[string][]Builtin{
 	// and the session's database search path.
 	"current_schemas": {
 		Builtin{
-			Types:        ArgTypes{{"unused", TypeBool}},
```

Comment:
Yes, the argument is there for Postgres compatibility (as is `current_schemas`). Is there a reason to remove it?